README.md

```diff
@@ -11,6 +11,23 @@ using the default hostname and ports: `localhost:9042`
 ```
 ./gradlew run
 ```
+
+## Running via docker-compose
+There is a docker-compose environment included in this repository. 
```

Comment:
For some reason spotless is breaking on this line ->
```suggestion
There is a docker-compose environment included in this repository.
```

Reply:
ah sorry hadn't run the build since I made the readme change. I assumed a README change wouldn't affect the build